pdb2pqr/biomolecule.py

```diff
@@ -482,7 +489,7 @@ def remove_hydrogens(self):
                 if atom.is_hydrogen:
                     residue.remove_atom(atom.name)
 
-    def assign_termini(self, chain, neutraln=False, neutralc=False):
+    def assign_termini(self, chain, *, neutraln=False, neutralc=False):
```

Comment:
It's not a wildcard. It's setting the boolean arguments as keyword argument only.

https://docs.astral.sh/ruff/rules/boolean-default-value-positional-argument/

https://peps.python.org/pep-3102/

Reply:
Why do we need it here?

Reply:
Because if users just write like `foo(True, False, True, False, False)` it's insanely difficult to read what it does. It's better to put especially boolean arguments as keyword arguments. If you mind I can revert of course.

Reply:
Oh, I understand now.  Thanks.

Reply:
Yeah, another advantage is that even if you add additional arguments to the function in the future, you don't really break the existing call of the function because the users would have explicitly written the name of the arguments :)